R/luz.R

```diff
@@ -0,0 +1,64 @@
+#' @rdname vetiver_create_description
+#' @export
+vetiver_create_description.luz_module_fitted <- function(model) {
+    glue("A luz module")
+}
+
+#' @rdname vetiver_create_meta
+#' @export
+vetiver_create_meta.luz_module_fitted <- function(model, metadata) {
+    pkgs <- c("luz", model$model$required_pkgs)
+    vetiver_meta(metadata, required_pkgs = pkgs)
+}
+
+#' @rdname vetiver_create_ptype
+#' @export
+vetiver_ptype.luz_module_fitted <- function(model, ...) {
```

Comment:
The best thing to do for now for the prototype is to take the [same approach as keras](https://github.com/rstudio/vetiver-r/blob/ad871b402586c3efa61735cf1d5cbc4a30777c07/R/keras.R#L16).